dotcom-rendering/src/components/Crossword.importable.tsx

```diff
@@ -0,0 +1,102 @@
+import { css } from '@emotion/react';
+import { Crossword as ReactCrossword } from '@guardian/react-crossword-next';
+import type { CrosswordProps } from '@guardian/react-crossword-next';
+import {
+	from,
+	headlineBold17,
+	palette,
+	space,
+	textSans12,
+	textSans14,
+} from '@guardian/source/foundations';
+import type { ReactNode } from 'react';
+import { memo } from 'react';
+
+const CluesHeader = memo(({ children }: { children: ReactNode }) => {
+	return (
+		<div
+			css={css`
+				${headlineBold17};
+				border-top: 1px solid ${palette.neutral[86]};
+				border-bottom: 1px dotted ${palette.neutral[86]};
+				background-color: ${palette.neutral[100]};
```

Comment:
Should we add these colours to the [palette](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/paletteDeclarations.ts) (assuming there's nothing there we can re-use?)